framework/console/controllers/QueueController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\console\controllers;
+
+use Yii;
+use yii\console\Controller;
+
+/**
+ * Runs the queue listener.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+class QueueController extends Controller
```

Comment:
```php
MyJob::listen();
```

MyJob listen what? It does not make sence. Queue usually contain **different** jobs (*Job classes). Worker run it step by step depends on queue (pool). Also note that job may fail, worker can create forks, worker can be daemon, etc. So worker technically is very complex component and we need provide it implementation from box.